scripts/install_flatbuffers.sh

```diff
@@ -23,6 +23,7 @@ then
           -DCMAKE_BUILD_TYPE=Release \
           -DCMAKE_CXX_COMPILER=$CXX \
           -DCMAKE_C_COMPILER=$CC \
+          -DCMAKE_CXX_FLAGS="-fPIC" \
```

Comment:
```suggestion
          -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
```

If the above works, we should prefer it.

Reply:
Just tried with FairMQ alfaci hosts and it appears to work.

Reply:
Changed, thanks